.github/workflows/ci-pipeline.yml

```diff
@@ -82,6 +82,42 @@ jobs:
         if: ${{ failure() }}
         run: find . -name config.log -exec cat {} \;
 
+  Build_static_u20:
```

Comment:
Is `_u20` clear enough at a glance? `Build_static_ubuntu20` could be clearer.

Reply:
![image](https://user-images.githubusercontent.com/9589374/186395523-882f0a52-22c2-4e81-a4b0-ecbad875f78c.png)

That name prefixes the job in the left column. The column width is fixed, the name needs to be short so that matrix parameters fit there (at least partly).

Reply:
Didn't remember it was fixed, OK, fair enough.